compiler/rustc_driver/src/lib.rs

```diff
@@ -1205,9 +1205,9 @@ pub fn report_ice(info: &panic::PanicInfo<'_>, bug_report_url: &str) {
     // If backtraces are enabled, also print the query stack
     let backtrace = env::var_os("RUST_BACKTRACE").map(|x| &x != "0").unwrap_or(false);
 
-    if backtrace {
-        TyCtxt::try_print_query_stack(&handler);
-    }
+    let num_frames = if backtrace { Some(2) } else { None };
```

Comment:
I think you got this inverted :laughing:

Reply:
oh shit, you are right, my bad, sorry🙈, I will fix

Reply:
so I guess if I fix it the others will be ok right? no need to add any change?

Reply:
probably not ^^